lib/takeaway.rb

```diff
@@ -0,0 +1,33 @@
+
+class Takeaway 
+attr_reader :menu, :payment
+
+    def initialize
+        @menu = { 'Salad'=>5, 'Pizza'=>8, 'Pasta'=>6 }
```

Comment:
Similar to how we extracted out a Station class from the journeys in last week's Oystercard pairing challenge, can you see a potential class we might extract out of here?